### ISSUE TITLE
Uncaught TypeError: R is not a function

### ISSUE THREAD

**Issue by strarsis** [marcusig/gutenberg-section-block#5]
@marcusig: The sections don't appear on the page in frontend at all.
Reloading the Gutenberg page editor results in empty page with JavaScript error logged to console:
````
Uncaught TypeError: R is not a function
    at save (blocks.build.js?ver=1:1)
    at Le (index.js?ver=1537825304:2)
    at De (index.js?ver=1537825304:2)
    at index.js?ver=1537825305:50
    at index.js?ver=1537825305:1
    at dt (index.js?ver=1537825304:2)
    at pt (index.js?ver=1537825304:2)
    at index.js?ver=1537825304:2
    at Array.reduce (<anonymous>)
    at index.js?ver=1537825304:2
react-dom.24169eaf.js:14804 The above error occurred in the <t> component:
    in t (created by r)
    in r (created by RemountOnPropChange(r))
    in RemountOnPropChange(r)
    in Unknown (created by WithDispatch(t))
    in WithDispatch(t)
    in Unknown (created by n)
    in n (created by RemountOnPropChange(n))
    in RemountOnPropChange(n)
    in WithSelect(Component)

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://fb.me/react-error-boundaries to learn more about error boundaries.
````

**Comment by kaha-digital** [comment]
This will be resolved - see issue #7

**Comment by marcusig** [comment]
This was fixed by @gubbigubbi.
